scanners/reactnative-expo/reactnative-expo.go

```diff
@@ -0,0 +1,665 @@
+package expo
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sync/atomic"
+	"time"
+
+	"github.com/bitrise-core/bitrise-init/models"
+	"github.com/bitrise-core/bitrise-init/scanners/android"
+	"github.com/bitrise-core/bitrise-init/scanners/cordova"
+	"github.com/bitrise-core/bitrise-init/scanners/ios"
+	"github.com/bitrise-core/bitrise-init/scanners/reactnative"
+	"github.com/bitrise-core/bitrise-init/steps"
+	"github.com/bitrise-core/bitrise-init/utility"
+	envmanModels "github.com/bitrise-io/envman/models"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	yaml "gopkg.in/yaml.v2"
+)
+
+// Name ...
+const Name = "react-native-expo"
+
+const (
+	workDirInputKey = "workdir"
+
+	gradleFileInputEnvKey = "GRADLE_BUILD_FILE_PATH"
```

Comment:
use the same options and configs as the ios and android scanner.

in the new android scanner we do not use `GRADLE_BUILD_FILE_PATH `